locales/es.json

```diff
@@ -36,5 +36,5 @@
   "Type to search": "Escribe para buscar",
   "No results for your search, try something different.": "No hay resultados para tu búsqueda, prueba algo diferente.",
   "JavaScript license information": "Información de licencia JavaScript",
-  "Toggle dark mode": "Cambiar a modo oscuro"
+  "Toggle dark mode": "Alternar a modo oscuro"
```

Comment:
`Alternar modo oscuro` is much better because this string says practically the same as before 😅